lib/msf/core/module/platform.rb

```diff
@@ -471,7 +471,13 @@ class NetBSD < Msf::Module::Platform
   class FreeBSD < Msf::Module::Platform
     Rank = 100
   end
-
+  #
+  #Empire
+  #
+  class Empire < Msf::Module::Platform
+    Rank = 100
```

Comment:
this is temporary right?

Reply:
Nope, I guess, we are taking `empire` as another platform to resolve few issues. It could have been included in windows, python, and macOS. But I thought it would rather be simpler if I put all of them into one platform.

Reply:
i mean the `Rank`

Reply:
Yeah, that's temporary.